.github/styles/dvc/admon.yml

```diff
@@ -0,0 +1,10 @@
+---
+extends: substitution
+message: Consider using '%s' instead of '%s'.
+ignorecase: true
+level: warning
+action:
+  name: replace
+swap:
+  'Note that' : an <admon> block
```

Comment:
@jorgeorpinel Here is an example custom rule that I added. Checks for `Not that` and suggest using `<admon>` block

Reply:
This is really exciting @daavoo!  It would be AMAZING if we could automate the grammar and style issues and free up reviewers time in this regard.  This would be a major leveraged win for all involved!